ui/components/app/assets/token-cell/cells/token-cell-title.tsx

```diff
@@ -27,7 +27,7 @@ export const TokenCellTitle = React.memo(
         <Tooltip
           position="bottom"
           html={token.title}
-          tooltipInnerClassName="multichain-token-list-item__tooltip"
```

Comment:
The class `multichain-token-list-item__tooltip` does not seem to exist.
Also, `tooltipInnerClassName` does not look like a property of `Tooltip`.

Reply:
nice find!